src/App.tsx

```diff
@@ -1,61 +1,131 @@
 import './App.scss';
+import { TodoList } from './components/TodoList';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import { FormEvent, useState } from 'react';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+type GetUserById = (todoId: number) => User | null;
+
+const getUserById: GetUserById = (todoId: number) => {
+  return usersFromServer.find(user => user.id === todoId) || null;
+};
+
+const getAllIds = (array: Todo[]) => {
+  const ids: number[] = [];
+
+  array.forEach(element => ids.push(element.id));
+
+  return ids;
+};
 
 export const App = () => {
+  const [todoList, setTodoList] = useState(todosFromServer);
+
+  const [title, setTitle] = useState('');
+  const [user, setUser] = useState(0);
+
+  const [titleError, setTitleError] = useState(false);
```

Comment:
use `enum` for ur errors
```suggestion
  const [titleError, setTitleError] = useState<ErrorType>(ErrorType.DEFAULT);
```
```
export enum ErrorType {
  TITLE_DEFAULT: '',
  TITLE_ERROR: 'Your errors text',
  .........
}
```